third_party/libdatachannel/scripts/build_libdatachannel.py

```diff
@@ -26,6 +34,27 @@ def main():
         "-DBUILD_SHARED_LIBS=OFF"
     ]
 
+    if target_cpu_type.startswith('arm'):
```

Comment:
This override the cmake_cmd set previously, suggest to handle both at the same time: 

```
elif target_cpu_type.startswith('x86'):
    cmake_cmd = [
        "cmake",
        "-B", "build",
        "-DUSE_GNUTLS=0",
        "-DUSE_NICE=0",
        "-DCMAKE_BUILD_TYPE=Release",
        "-DCMAKE_CXX_FLAGS=-Wno-shadow",
        "-DBUILD_SHARED_LIBS=OFF"
    ]
```

Reply:
Thank you for your review, We fixed your comments